apps/frontend/src/modules/projects/slices/actions.ts

```diff
@@ -24,13 +25,18 @@ const getById = createAsyncThunk<
 });
 
 const loadAll = createAsyncThunk<
-	ProjectGetAllResponseDto,
-	string | undefined,
+	{ page: number } & ProjectGetAllResponseDto,
+	ProjectGetAllRequestDto,
 	AsyncThunkConfig
 >(`${sliceName}/load-all`, async (query, { extra }) => {
 	const { projectApi } = extra;
 
-	return await projectApi.getAll(query);
+	const result = await projectApi.getAll(query);
+
+	return {
+		...result,
+		page: query.page,
```

Comment:
looks like we don't need to return this as result of action, we can access page as action arguments inside reducer
```suggestion
```

Reply:
like this?
```
const { page } = action.meta.arg
```